mlir/include/mlir/Dialect/Transform/IR/TransformDialect.td

```diff
@@ -42,6 +42,14 @@ def Transform_Dialect : Dialect {
       constexpr const static ::llvm::StringLiteral
           kArgReadOnlyAttrName = "transform.readonly";
 
+      /// Above attribute names as `StringAttr`.
```

Comment:
Nit: this comment is rendered, e.g., for autocompletion, where there is no "above" context. Prefer something more explanatory.